mathics_scanner/data/named-characters.yml

```diff
@@ -1827,6 +1834,7 @@ Cup:
   wl-unicode-name: SMILE
 
 CupCap:
+  amslatex: '\overset{\smile}{\frown}'
```

Comment:
same as above.

Reply:
To find these amslatex form, I am evaluating in WMA expressions like `a [\CupCap] a //TeXForm` and picking the representation from there.